hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -37,10 +37,23 @@ objects:
     name: aws-vpce-operator
   spec:
     clusterDeploymentSelector:
-      matchLabels:
-        api.openshift.com/managed: 'true'
-        api.openshift.com/fedramp: 'true'
-        api.openshift.com/private-link: 'true'
+      matchExpressions:
+        - key: api.openshift.com/managed
+          operator: In
+          values:
+            - "true"
+        - key: api.openshift.com/fedramp
+          operator: In
+          values:
+            - "true"
+        - key: api.openshift.com/private-link
+          operator: In
+          values:
+            - "true"
+        - key: ext-managed.openshift.io/hive-shard=true
```

Comment:
This is 100% my typo, this should be `- key: ext-managed.openshift.io/hive-shard`
I accidentally pasted the exact label from the cluster in my sample haha.

Reply:
Great catch